modules/nix/linux-builder.nix

```diff
@@ -20,6 +22,7 @@ let
     rm -rf $TMPDIR
     mkdir -p $TMPDIR
     trap "rm -rf $TMPDIR" EXIT
+    rm -f ${WorkingDirectory}/nixos.qcow2
```

Comment:
If you could also make an option that controls this, possibly named something like `wipeOnRestart` or `ephemeral`

Reply:
Done.